db/migrate/20210805124942_remove_report_from_stats_reports.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveReportFromStatsReports < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :stats_reports, :report, :string, after: :report_name
```

Comment:
I was hoping that `after: :report_name` would make a `db:rollback` put the field back in the same place in the database table but apparently it doesn't. :-(

Reply:
never heard of an `after:` option. do you have a link to its documentation?

Reply:
Hmmm, it appears that `after` is a MySQL extension and it works with the `mysql` adapter but not `pg`.